clipboard-jh package: Opening libcbx11.so to look for setX11Clipboard failed, aborting operation

### Describe the bug
The [Clipboard package](https://github.com/NixOS/nixpkgs/tree/nixos-unstable/pkgs/tools/misc/clipboard-jh) simply doesn't work for me. The problem seems to be in `cb` binary not being able to find Clipboard's own libraries. When running in debug mode some useful information comes out:

```console
$ ./bin/cb copy "dsds"
✅ Copied text "dsds"
Opening libcbx11.so to look for setX11Clipboard failed, aborting operation
Specific error: libcbx11.so: cannot open shared object file: No such file or directory
Setting X11 clipboard failed, trying Wayland
Opening libcbwayland.so to look for setWaylandClipboard failed, aborting operation
Specific error: libcbwayland.so: cannot open shared object file: No such file or directory
Setting Wayland clipboard failed
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/gui/CMakeFiles/gui.dir/src/infermime.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/gui/CMakeFiles/gui.dir/src/utils.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/gui/CMakeFiles/gui.dir/src/gui.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/gui/CMakeFiles/gui.dir/src/fork.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/cb/CMakeFiles/cb.dir/src/clipboard.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/cb/CMakeFiles/cb.dir/src/actions.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/cb/CMakeFiles/cb.dir/src/messages.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/cb/CMakeFiles/cb.dir/src/themes.cpp.gcda:Skip
profiling:/build:Cannot create directory
profiling:/build/93rzrplx201b09v308sw5r0hwp0q6g6l-source/build/src/cb/CMakeFiles/cb.dir/src/x11wl.cpp.gcda:Skip
```

However, those libraries are physically present:
```console
dr-xr-xr-x     - root  1 янв  1970  .
dr-xr-xr-x     - root  1 янв  1970  ├── bin
.r-xr-xr-x  1,0M root  1 янв  1970  │  └── cb
dr-xr-xr-x     - root  1 янв  1970  ├── lib
.r-xr-xr-x  690k root  1 янв  1970  │  ├── libcbwayland.so
.r-xr-xr-x  455k root  1 янв  1970  │  └── libcbx11.so
dr-xr-xr-x     - root  1 янв  1970  └── share
dr-xr-xr-x     - root  1 янв  1970     └── man
dr-xr-xr-x     - root  1 янв  1970        └── man1
.r--r--r--  1,3k root  1 янв  1970           └── cb.1.gz
```

I also tried to modify `LD_LIBRARY_PATH` to point to the lib directory, that didn't help either.

### Steps To Reproduce
1. Try to do literally anything with `cb`, like copy and paste something.

### Notify maintainers

Notifying @dit7ya as they are a maintainer.

### Metadata

clipboard-jh version: 0.6.0, [commit](https://github.com/NixOS/nixpkgs/commit/eb374d7bae42d185422d7f4bbcd4ed0a5f048e10).

Xorg, KDE Plasma

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.108, NixOS, 23.05 (Stoat), 23.05pre478022.d6b863fd9b7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(root): `"home-manager, nixos, nixos-stable-22.11, nixos-unstable"`
 - channels(heinwol): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

<details>
  <summary>And here is the full log</summary>
  
  ```log
$ nix build --log-format bar-with-logs
warning: Git tree '/home/heinwol/temp/Clipboard' is dirty
Clipboard> unpacking sources
Clipboard> unpacking source archive /nix/store/aqssgmz0kahklsxyvpbj864jzpi33cm1-cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source
Clipboard> source root is cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source
Clipboard> patching sources
Clipboard> configuring
Clipboard> fixing cmake files...
Clipboard> cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/share/doc/Clipboard -DCMAKE_INSTALL_INFODIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/b5mkki8bysygaf7a01jpj88zrxibkrv4-gcc-wrapper-12.2.0/bin/strip -DCMAKE_RANLIB=/nix/store/b5mkki8bysygaf7a01jpj88zrxibkrv4-gcc-wrapper-12.2.0/bin/ranlib -DCMAKE_AR=/nix/store/b5mkki8bysygaf7a01jpj88zrxibkrv4-gcc-wrapper-12.2.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0 -DCMAKE_BUILD_TYPE=Debug
Clipboard> -- The CXX compiler identification is GNU 12.2.0
Clipboard> -- The C compiler identification is GNU 12.2.0
Clipboard> -- Detecting CXX compiler ABI info
Clipboard> -- Detecting CXX compiler ABI info - done
Clipboard> -- Check for working CXX compiler: /nix/store/b5mkki8bysygaf7a01jpj88zrxibkrv4-gcc-wrapper-12.2.0/bin/g++ - skipped
Clipboard> -- Detecting CXX compile features
Clipboard> -- Detecting CXX compile features - done
Clipboard> -- Detecting C compiler ABI info
Clipboard> -- Detecting C compiler ABI info - done
Clipboard> -- Check for working C compiler: /nix/store/b5mkki8bysygaf7a01jpj88zrxibkrv4-gcc-wrapper-12.2.0/bin/gcc - skipped
Clipboard> -- Detecting C compile features
Clipboard> -- Detecting C compile features - done
Clipboard> -- Building the Clipboard Project in Debug mode
Clipboard> -- Looking for fork
Clipboard> -- Looking for fork - found
Clipboard> -- Found X11: /nix/store/jbc3f1agds5a6d0yas6i30hwr36al8jd-xorgproto-2021.5/include
Clipboard> -- Looking for XOpenDisplay in /nix/store/pciixhi1sl4rc4b7wk705qzq8461l14h-libX11-1.8.4/lib/libX11.so
Clipboard> -- Looking for XOpenDisplay in /nix/store/pciixhi1sl4rc4b7wk705qzq8461l14h-libX11-1.8.4/lib/libX11.so - found
Clipboard> -- Looking for gethostbyname
Clipboard> -- Looking for gethostbyname - found
Clipboard> -- Looking for connect
Clipboard> -- Looking for connect - found
Clipboard> -- Looking for remove
Clipboard> -- Looking for remove - found
Clipboard> -- Looking for shmat
Clipboard> -- Looking for shmat - found
Clipboard> -- Building the Clipboard Project with X11 support
Clipboard> -- Found PkgConfig: /nix/store/985iynvd47l5y3mpj0fx30s3cip6isix-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
Clipboard> -- Checking for modules 'wayland-client;wayland-protocols'
Clipboard> --   Found wayland-client, version 1.22.0
Clipboard> --   Found wayland-protocols, version 1.31
Clipboard> -- Building the Clipboard Project with Wayland support
Clipboard> -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
Clipboard> -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
Clipboard> -- Found Threads: TRUE
Clipboard> -- Configuring done
Clipboard> -- Generating done
Clipboard> CMake Warning:
Clipboard>   Manually-specified variables were not used by the project:
Clipboard>     BUILD_TESTING
Clipboard>     CMAKE_EXPORT_NO_PACKAGE_REGISTRY
Clipboard>     CMAKE_INSTALL_BINDIR
Clipboard>     CMAKE_INSTALL_DOCDIR
Clipboard>     CMAKE_INSTALL_INCLUDEDIR
Clipboard>     CMAKE_INSTALL_INFODIR
Clipboard>     CMAKE_INSTALL_LIBEXECDIR
Clipboard>     CMAKE_INSTALL_LOCALEDIR
Clipboard>     CMAKE_INSTALL_MANDIR
Clipboard>     CMAKE_INSTALL_OLDINCLUDEDIR
Clipboard>     CMAKE_INSTALL_SBINDIR
Clipboard>     CMAKE_POLICY_DEFAULT_CMP0025
Clipboard>
Clipboard> -- Build files have been written to: /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/build
Clipboard> cmake: enabled parallel building
Clipboard> cmake: enabled parallel installing
Clipboard> building
Clipboard> build flags: -j8 SHELL=/nix/store/j80wcbm6bb1nsqsgmniq43rg980p27q9-bash-5.2-p15/bin/bash
Clipboard> [  5%] Building CXX object src/gui/CMakeFiles/gui.dir/src/gui.cpp.o
Clipboard> [  5%] Building CXX object src/gui/CMakeFiles/gui.dir/src/utils.cpp.o
Clipboard> [  8%] Building CXX object src/gui/CMakeFiles/gui.dir/src/fork.cpp.o
Clipboard> [ 11%] Generating generated/include/wayland-xdg-shell.hpp
Clipboard> [ 14%] Building CXX object src/gui/CMakeFiles/gui.dir/src/infermime.cpp.o
Clipboard> [ 17%] Building CXX object src/gui/CMakeFiles/gui.dir/src/x11wl/mime.cpp.o
Clipboard> [ 17%] Built target cbwayland_generatedheaders
Clipboard> [ 20%] Linking CXX static library ../../libgui.a
Clipboard> [ 20%] Built target gui
Clipboard> [ 22%] Generating generated/src/wayland-xdg-shell.c
Clipboard> [ 25%] Building CXX object src/cb/CMakeFiles/cb.dir/src/actions.cpp.o
Clipboard> [ 34%] Building CXX object src/cbx11/CMakeFiles/cbx11.dir/src/x11.cpp.o
Clipboard> [ 34%] Building CXX object src/cb/CMakeFiles/cb.dir/src/messages.cpp.o
Clipboard> [ 34%] Building CXX object src/cb/CMakeFiles/cb.dir/src/clipboard.cpp.o
Clipboard> [ 37%] Building CXX object src/cb/CMakeFiles/cb.dir/src/themes.cpp.o
Clipboard> [ 40%] Building CXX object src/cb/CMakeFiles/cb.dir/src/x11wl.cpp.o
Clipboard> [ 45%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/wayland.cpp.o
Clipboard> [ 45%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/fd.cpp.o
Clipboard> [ 48%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/buffer.cpp.o
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp: In member function 'X11Window X11Connection::createWindow()':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::background_pixmap' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard>   474 |     XSetWindowAttributes attributes {.event_mask = PropertyChangeMask};
Clipboard>       |                                                                      ^
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::background_pixel' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::border_pixmap' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::border_pixel' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::bit_gravity' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::win_gravity' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::backing_store' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::backing_planes' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::backing_pixel' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::save_under' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::do_not_propagate_mask' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::override_redirect' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::colormap' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:474:70: warning: missing initializer for member 'XSetWindowAttributes::cursor' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cb/src/actions.cpp: In function 'void PerformAction::status()':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cb/src/actions.cpp:439:23: warning: comparison of integer expressions of different signedness: 'int' and 'long unsigned int' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare-Wsign-compare8;;]
Clipboard>   439 |     for (int i = 0; i < longestClipboardLength + 1; i++)
Clipboard>       |                     ~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cb/src/actions.cpp:444:23: warning: comparison of integer expressions of different signedness: 'int' and 'long unsigned int' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare-Wsign-compare8;;]
Clipboard>   444 |     for (int i = 0; i < cols - 2; i++)
Clipboard>       |                     ~~^~~~~~~~~~
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cb/src/actions.cpp: In function 'void PerformAction::info()':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cb/src/actions.cpp:564:23: warning: comparison of integer expressions of different signedness: 'int' and 'long unsigned int' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare-Wsign-compare8;;]
Clipboard>   564 |     for (int i = 0; i < cols - 2; i++)
Clipboard>       |                     ~~^~~~~~~~~~
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp: In member function 'void X11Window::setEventMask(long int)':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::background_pixmap' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard>   932 |     XSetWindowAttributes attributes = {.event_mask = eventMask};
Clipboard>       |                                                               ^
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::background_pixel' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::border_pixmap' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::border_pixel' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::bit_gravity' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::win_gravity' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::backing_store' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::backing_planes' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::backing_pixel' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::save_under' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::do_not_propagate_mask' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::override_redirect' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::colormap' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:932:63: warning: missing initializer for member 'XSetWindowAttributes::cursor' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp: In static member function 'static XEvent X11SelectionDaemon::makeSelectionNotify(const XSelectionRequestEvent&)':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:1060:14: warning: missing initializer for member 'XSelectionEvent::serial' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard>  1060 |             }};
Clipboard>       |              ^
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbx11/src/x11.cpp:1060:14: warning: missing initializer for member 'XSelectionEvent::send_event' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wmissing-field-initializers-Wmissing-field-initializers8;;]
Clipboard> [ 51%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/callback.cpp.o
Clipboard> [ 54%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/data_device.cpp.o
Clipboard> [ 57%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/data_offer.cpp.o
Clipboard> [ 60%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/data_source.cpp.o
Clipboard> [ 62%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/display.cpp.o
Clipboard> In file included from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_offer.hpp:18,
Clipboard>                  from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_offer.cpp:15:
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_data_offer*, unsigned int}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_offer.cpp:20:27:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard>   177 | void noHandler(args_t... args) {}
Clipboard>       |                ~~~~~~^~~~~~~~
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> In file included from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_device.hpp:18,
Clipboard>                  from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_device.cpp:15:
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_data_device*, unsigned int, wl_surface*, int, int, wl_data_offer*}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_device.cpp:22:18:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard>   177 | void noHandler(args_t... args) {}
Clipboard>       |                ~~~~~~^~~~~~~~
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#3' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#4' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#5' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#6' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_data_device*}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_device.cpp:23:18:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_data_device*, unsigned int, int, int}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_device.cpp:24:19:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#3' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#4' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> [ 65%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/keyboard.cpp.o
Clipboard> [ 68%] Linking CXX shared module ../../libcbx11.so
Clipboard> [ 71%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/registry.cpp.o
Clipboard> In file included from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_source.hpp:19,
Clipboard>                  from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_source.cpp:15:
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_data_source*, const char*}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_source.cpp:21:19:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard>   177 | void noHandler(args_t... args) {}
Clipboard>       |                ~~~~~~^~~~~~~~
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_data_source*}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_source.cpp:24:31:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_data_source*, unsigned int}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/data_source.cpp:26:19:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> [ 74%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/seat.cpp.o
Clipboard> [ 77%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/shm.cpp.o
Clipboard> [ 80%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/shm_pool.cpp.o
Clipboard> [ 82%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/surface.cpp.o
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.cpp: In member function 'void WlKeyboard::onLeave(uint32_t, wl_surface*)':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.cpp:58:40: warning: unused parameter 'serial' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard>    58 | void WlKeyboard::onLeave(std::uint32_t serial, wl_surface* surface) {
Clipboard>       |                          ~~~~~~~~~~~~~~^~~~~~
Clipboard> [ 85%] Linking CXX executable ../../cb
Clipboard> In file included from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.hpp:18,
Clipboard>                  from /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.cpp:15:
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_keyboard*, unsigned int, int, unsigned int}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.cpp:19:19:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard>   177 | void noHandler(args_t... args) {}
Clipboard>       |                ~~~~~~^~~~~~~~
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#3' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#4' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_keyboard*, unsigned int, unsigned int, unsigned int, unsigned int}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.cpp:22:16:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#3' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#4' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#5' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_keyboard*, unsigned int, unsigned int, unsigned int, unsigned int, unsigned int}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.cpp:23:22:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#3' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#4' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#5' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#6' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp: In instantiation of 'void noHandler(args_t ...) [with args_t = {void*, wl_keyboard*, int, int}]':
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/keyboard.cpp:24:24:   required from here
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#0' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#1' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#2' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> /build/cnq3jdjkn9s19cd19pvyxvfby4d0d93z-source/src/cbwayland/src/objects/spec.hpp:177:22: warning: unused parameter 'args#3' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-parameter-Wunused-parameter8;;]
Clipboard> [ 88%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/xdg_surface.cpp.o
Clipboard> [ 91%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/xdg_toplevel.cpp.o
Clipboard> [ 91%] Built target cbx11
Clipboard> [ 94%] Building CXX object src/cbwayland/CMakeFiles/cbwayland.dir/src/objects/xdg_wm_base.cpp.o
Clipboard> [ 97%] Building C object src/cbwayland/CMakeFiles/cbwayland.dir/generated/src/wayland-xdg-shell.c.o
Clipboard> [100%] Linking CXX shared module ../../libcbwayland.so
Clipboard> [100%] Built target cb
Clipboard> [100%] Built target cbwayland
Clipboard> installing
Clipboard> install flags: -j8 SHELL=/nix/store/j80wcbm6bb1nsqsgmniq43rg980p27q9-bash-5.2-p15/bin/bash install
Clipboard> [  2%] Built target cbwayland_generatedheaders
Clipboard> [ 20%] Built target gui
Clipboard> [ 25%] Built target cbx11
Clipboard> [ 42%] Built target cb
Clipboard> [100%] Built target cbwayland
Clipboard> Install the project...
Clipboard> -- Install configuration: "Debug"
Clipboard> -- Installing: /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib/libcbx11.so
Clipboard> -- Installing: /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib/libcbwayland.so
Clipboard> -- Installing: /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/bin/cb
Clipboard> -- Set runtime path of "/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/bin/cb" to "/nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0//nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib"
Clipboard> -- Installing: /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/share/man/man1/cb.1
Clipboard> post-installation fixup
Clipboard> shrinking RPATHs of ELF executables and libraries in /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0
Clipboard> shrinking /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/bin/cb
Clipboard> shrinking /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib/libcbx11.so
Clipboard> shrinking /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib/libcbwayland.so
Clipboard> checking for references to /build/ in /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0...
Clipboard> gzipping man pages under /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/share/man/
Clipboard> patching script interpreter paths in /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0
Clipboard> stripping (with command strip and flags -S -p) in  /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/lib /nix/store/z34dj58v8fsqv4kxqkqnfxrv90bflj66-Clipboard-0.7.0/bin
  ```
</details>

--- a/pkgs/tools/misc/clipboard-jh/default.nix
+++ b/pkgs/tools/misc/clipboard-jh/default.nix
@@ -4,6 +4,7 @@
 , cmake
 , libffi
 , pkg-config
+, patchelf
 , wayland-protocols
 , wayland
 , xorg
@@ -44,6 +45,10 @@ stdenv.mkDerivation rec {
     "-Wno-dev"
     "-DINSTALL_PREFIX=${placeholder "out"}"
   ];
+
+  postFixup = lib.optionalString stdenv.isLinux ''
+    patchelf $out/bin/cb --add-rpath $out/lib
+  '';
 
   meta = with lib; {
     description = "Cut, copy, and paste anything, anywhere, all from the terminal";

